swift/ios-shoppe-demo/Resources/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>FSOrgId</key>
+	<string>KWH</string>
```

Comment:
Can we stub this? Like a `YOUR_ORG_HERE`.  This way we don't commit your org to public GH.